addons/initialise-sprite-position/userscript.js

```diff
@@ -0,0 +1,22 @@
+export default async function ({ addon }) {
+  let vm = addon.tab.traps.vm;
+  let oldAddSprite = vm.constructor.prototype.addSprite;
+  
+  vm.constructor.prototype.addSprite = function (input) {
+    let spriteObj;
+    spriteObj = JSON.parse(input);
+    let isEmpty = spriteObj.costumes[0].baseLayerMD5 === "cd21514d0531fdffb22204e0ec5ed84a.svg";
+      
+    if (isEmpty || !addon.settings.get("library")) {
+      if (spriteObj.scratchX) {
+        spriteObj.scratchX = +addon.settings.get("x");
+        spriteObj.scratchY = +addon.settings.get("y");
+      }
+      if (spriteObj.x) {
+        spriteObj.x = +addon.settings.get("x");
+        spriteObj.y = +addon.settings.get("y");
+      }
+    }
+    oldAddSprite.call(this, JSON.stringify(spriteObj));
```

Comment:
```suggestion
    return oldAddSprite.call(this, JSON.stringify(spriteObj));
```
will probably fix imports never ending

Reply:
Thanks, I think I fixed it another way (unpushed) but I'll add that in as a reserve. Thanks!